addon/ng2/blueprints/pipe/files/src/app/pipes/__name__/__name__.ts

```diff
@@ -7,7 +7,7 @@ import {Pipe} from 'angular2/angular2';
 export class <%= classifiedModuleName %> {
 
   transform(value, args?) {
-    return value;
+    return !value;
```

Comment:
oh I see. you are trying to make the test pass. how about returning null and updating the test to that too